tests/test_report.py

```diff
@@ -0,0 +1,128 @@
+#             # emulates how this project would be run as a module. The only thing
+#             # being done by __main__ is importing the main entrypoint of the
+#             # package and running it, so there is nothing to use from this
+#             # import. As a result, we can safely ignore this warning.
+#             # Third-Party Libraries
+#             import report.__main__  # noqa: F401
+#     captured = capsys.readouterr()
+#     assert (
+#         captured.out == f"{PROJECT_VERSION}\n"
+#     ), "standard output by '--version' should agree with module.__version__"
+
+
+@pytest.mark.skipif(
+    RELEASE_TAG in [None, ""], reason="this is not a release (RELEASE_TAG not set)"
+)
+def test_release_version():
+    """Verify that release tag version agrees with the module version."""
+    assert (
+        RELEASE_TAG == f"v{PROJECT_VERSION}"
+    ), "RELEASE_TAG does not match the project version"
+
+
+@pytest.mark.parametrize("level", log_levels)
+def test_log_levels(level):
+    """Validate commandline log-level arguments."""
+    with patch.object(sys, "argv", ["bogus", f"--log-level={level}", "1", "1"]):
+        with patch.object(logging.root, "handlers", []):
+            assert (
+                logging.root.hasHandlers() is False
+            ), "root logger should not have handlers yet"
+            return_code = report.main()
+            assert (
+                logging.root.hasHandlers() is True
+            ), "root logger should now have a handler"
+            assert return_code == 0, "main() should return success (0)"
+
+
+def test_bad_log_level():
+    """Validate bad log-level argument returns error."""
+    with patch.object(sys, "argv", ["bogus", "--log-level=emergency", "1", "1"]):
+        return_code = report.main()
+        assert return_code == 1, "main() should return failure"
+
+
+@pytest.mark.parametrize("dividend, divisor, quotient", div_params)
+def test_division(dividend, divisor, quotient):
+    """Verify division results."""
+    result = report.example_div(dividend, divisor)
+    assert result == quotient, "result should equal quotient"
+
+
+@pytest.mark.slow
+def test_slow_division():
+    """Example of using a custom marker.
+
+    This test will only be run if --runslow is passed to pytest.
+    Look in conftest.py to see how this is implemented.
+    """
+    # Standard Python Libraries
+    import time
+
+    result = report.example_div(256, 16)
+    time.sleep(4)
+    assert result == 16, "result should equal be 16"
+
+
+def test_zero_division():
+    """Verify that division by zero throws the correct exception."""
+    with pytest.raises(ZeroDivisionError):
+        report.example_div(1, 0)
+
+
+def test_zero_divisor_argument():
+    """Verify that a divisor of zero is handled as expected."""
+    with patch.object(sys, "argv", ["bogus", "1", "0"]):
+        return_code = report.main()
+        assert return_code == 1, "main() should exit with error"
```

Comment:
These skeleton tests can be deleted and should be replaced with some useful tests for this project.
```suggestion
```